docs/Installation.md

```diff
@@ -591,7 +583,6 @@ path:
 
 ```sh
 sed -i '' '/[[:<:]]engine[[:>:]]/ s/=.*/= SQLite/' /usr/local/etc/zonemaster/backend_config.ini
-sed -i '' '/[[:<:]]database_name[[:>:]]/ s:=.*:= /var/db/zonemaster/db.sqlite:' /usr/local/etc/zonemaster/backend_config.ini

```

Comment:
In FreeBSD, the location of the SQLite database file should be /var/db/zonemaster, not /var/lib/zonemaster.

Reply:
Good catch! I'll add that line back.